db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql

```diff
@@ -6,14 +6,14 @@ CREATE TABLE `groups_contest_items` (
     `can_enter_until` datetime NOT NULL DEFAULT '9999-12-31 23:59:59' COMMENT 'Time until which the group can “enter” this contest',
     `additional_time` time NOT NULL DEFAULT '00:00:00' COMMENT 'Time that was attributed (can be negative) to this group for this contest',
     PRIMARY KEY (`group_id`, `item_id`)
-);
+) COMMENT 'Per-group constraints on contests participation';
 
 CREATE TABLE `contest_participations` (
     `group_id` bigint(20) NOT NULL,
     `item_id` bigint(20) NOT NULL,
     `entered_at` datetime DEFAULT NULL COMMENT 'Time at which the group entered the contest',
     PRIMARY KEY (`group_id`, `item_id`)
-);
+) COMMENT 'Connections between contests and contestant groups';
```

Comment:
Not clear what could "connections" be. Relations?
"Team or user information when participating to contests" ? As the fact it is related "end-members" (team or user) is the main differentiator from "contests participation".

Reply:
fixed